exporter/googlecloudexporter/go.mod

```diff
@@ -46,7 +46,7 @@ require (
 	go.opentelemetry.io/otel/sdk v1.11.1 // indirect
 	go.opentelemetry.io/otel/trace v1.11.1 // indirect
 	go.uber.org/atomic v1.10.0 // indirect
-	go.uber.org/zap v1.23.0 // indirect
```

Comment:
Was this a result of a `go mod tidy`? Curious why this changed.

Reply:
Yeah, it was a result of go mod tidy. I'm guessing go mod tidy wasn't used in the previous pr that updated the google exporter.